acapy_plugin_toolbox/invitations.py

```diff
@@ -0,0 +1,186 @@
+    ConnectionInvitation,
+)
+from aries_cloudagent.protocols.problem_report.message import ProblemReport
+from aries_cloudagent.storage.error import StorageNotFoundError
+from aries_cloudagent.messaging.valid import INDY_ISO8601_DATETIME
+
+from .util import generate_model_schema, admin_only
+
+PROTOCOL = 'https://github.com/hyperledger/aries-toolbox/tree/master/docs/admin-invitations/0.1'
+
+# Message Types
+INVITATION_GET_LIST = '{}/invitation-get-list'.format(PROTOCOL)
+INVITATION_LIST = '{}/invitation-list'.format(PROTOCOL)
+CREATE_INVITATION = '{}/create-invitation'.format(PROTOCOL)
+INVITATION = '{}/invitation'.format(PROTOCOL)
+
+# Message Type string to Message Class map
+MESSAGE_TYPES = {
+    CREATE_INVITATION:
+        'acapy_plugin_toolbox.invitations'
+        '.CreateInvitation',
+    INVITATION_GET_LIST:
+        'acapy_plugin_toolbox.invitations'
+        '.InvitationGetList',
+    INVITATION:
+        'acapy_plugin_toolbox.invitations'
+        '.Invitation',
+}
+
+
+async def setup(context: InjectionContext):
+    """Setup the connections plugin."""
+    protocol_registry = await context.inject(ProtocolRegistry)
+    protocol_registry.register_message_types(
+        MESSAGE_TYPES
+    )
+
+
+InvitationGetList, InvitationGetListSchema = generate_model_schema(
+    name='InvitationGetList',
+    handler='acapy_plugin_toolbox.invitations.InvitationGetListHandler',
+    msg_type=INVITATION_GET_LIST,
+    schema={
+
+    }
+)
+
+CreateInvitation, CreateInvitationSchema = generate_model_schema(
+    name='CreateInvitation',
+    handler='acapy_plugin_toolbox.invitations.CreateInvitationHandler',
+    msg_type=CREATE_INVITATION,
+    schema={
+        'label': fields.Str(required=False),
+        'alias': fields.Str(required=False), #default?
+        'role': fields.Str(required=False),
+        'auto_accept': fields.Boolean(missing=False),
+        'multi_use': fields.Boolean(missing=False),
+    }
+)
+
+Invitation, InvitationSchema = generate_model_schema(
+    name='Invitation',
+    handler='acapy_plugin_toolbox.util.PassHandler',
+    msg_type=INVITATION,
+    schema={
+        'id': fields.Str(required=True),
+        'label': fields.Str(required=False),
+        'alias': fields.Str(required=False), #default?
+        'role': fields.Str(required=False),
+        'auto_accept': fields.Boolean(missing=False),
+        'multi_use': fields.Boolean(missing=False),
+        'invitation_url': fields.Str(required=True),
+        'created_date': fields.Str(
+            required=False, description="Time of record creation", **INDY_ISO8601_DATETIME
+        ),
+    }
+)
+
+Connection, ConnectionSchema = generate_model_schema(
+    name='Connection',
+    handler='acapy_plugin_toolbox.util.PassHandler',
+    msg_type="/connection", #message not used, so this doesn't matter
+    schema={
+        'connection': fields.Nested(ConnectionRecordSchema, required=True),
+    }
+)
```

Comment:
Seems we can just delete this?

Reply:
Good catch. It was needed before, but not after my schema refactor